README.md

```diff
@@ -2,11 +2,24 @@
 
 This is Thoth SLO Reporter to share its achievements and behaviour with the outside world.
 
+# Adding a new SLI report
+1. Add a class under [thoth/slo_reporter](https://github.com/thoth-station/slo-reporter/tree/master/thoth/slo_reporter) if it doesn't exist for the kind of report. This class would inherit the base class `SLIBase` from [sli_base.py](https://github.com/thoth-station/slo-reporter/blob/master/thoth/slo_reporter/sli_base.py). 
+2. Add an HTML jinja template that is to be included in the report here - [Link](https://github.com/thoth-station/slo-reporter/tree/master/thoth/slo_reporter/static/templates)
+3. Add a method to load the template you designed in [sli_template.py](https://github.com/thoth-station/slo-reporter/blob/master/thoth/slo_reporter/sli_template.py) and passes down the parameters and passes them to the template. 
+4. The query method can be tested against the Prometheus web UI before being added to the method here - [Link](https://prometheus-dh-prod-monitoring.cloud.datahub.psi.redhat.com/graph)
+5. In the class that you created in step 1, add the `aggregate_info` method to return the query and the report.
+    ```python    
+    def _aggregate_info(self):
+        """"Aggregate info required for knowledge graph SLI Report."""
+        return {"query": self._query_sli(), "report_method": self._report_sli}
+    ```
+6. Remember to import the class in [sli_report.py](https://github.com/thoth-station/slo-reporter/blob/master/thoth/slo_reporter/sli_report.py) and add it to the `REPORT_SLI_CONTEXT` dictionary. 
```

Comment:
Let's mention that the order in which they add new report to `class SLIReport` matters

Therefore when they insert the new context the order need to be decided as well. I would put Kebechet after Thoth Learning, because users of Thoth can see what we collect respect to the Python world and then we show the integrations then we go to user-api which is behind all integrations of Thoth and then we show the workflow which run in the background.

    REPORT_SLI_CONTEXT = {
        SLIPyPIKnowledgeGraph._SLI_NAME: SLIPyPIKnowledgeGraph()._aggregate_info(),
        SLIKnowledgeGraph._SLI_NAME: SLIKnowledgeGraph()._aggregate_info(),
        SLILearning._SLI_NAME: SLILearning()._aggregate_info(),
        SLIUserAPI._SLI_NAME: SLIUserAPI()._aggregate_info(),
        SLIWorkflowQuality._SLI_NAME: SLIWorkflowQuality()._aggregate_info(),
    }